### DISCUSSION TITLE
Общие шаблоны slash commands

### DISCUSSION THREAD

**Discussion by Amiditin** [atls/convention#48] [Ideas]
Проблема: у членов команды разный стиль написания статусов на конец дня или оформления PR. Как, например, эта задача https://github.com/torin-asakura/shdvor/issues/148.

Предлагаю в конвенции [Настройка Github](https://github.com/atls/convention/wiki/%D0%A2%D1%80%D0%B5%D0%B1%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F-%D0%BA-%D0%BE%D0%BA%D1%80%D1%83%D0%B6%D0%B5%D0%BD%D0%B8%D1%8E#%D0%BD%D0%B0%D1%81%D1%82%D1%80%D0%BE%D0%B9%D0%BA%D0%B0-github) добавить пункт про настройку [slash-commands](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/about-slash-commands), с нашими шаблонами. Каждый член команды должен будет добавить наши шаблоны и использовать их через `saved replies`, примеры:

<details><summary>Статус на конец дня</summary>

```markdown
#### Что сделано:
-
#### Что дальше:
-
#### Актуальная ветка:
- 
```
</details> 

<details><summary>Дейлик</summary>

```markdown
#### Какие задачи выполнял вчера? Укажи #issues и сообщи в каком они состоянии
-
#### Какие задачи будешь делать сегодня? Укажи #issues
-
#### Что тебя блокирует? (Этот пункт используется когда тебя что-то блокирует)
-
#### Есть ли личные дела из-за которых нужно отсутствовать на рабочем месте в течение рабочего дня? (Этот пункт используется когда дела есть)
10:00 - 12:00 мск
```
</details>

<details><summary>Pull Request</summary>

```markdown
Resolves: задача

<details>
<summary>Пруф</summary>


</details>
```
</details>

**Comment by Amiditin** [comment]
@Nelfimov @TorinAsakura

**Reply by TorinAsakura** [reply]
Мощь! Давно такую хрень хотел, но, даже не знал, что она в гитхабе уже завезена! Полностью поддерживаю!

**Reply by Nelfimov** [reply]
Скринкаст

https://github.com/user-attachments/assets/0cacace4-fd4c-43bd-94ea-2ef05878ce9c

@TorinAsakura подтверди пожалуйста шаблоны

**Reply by TorinAsakura** [reply]
- daily у нас сейчас есть в ДСМе
- текущий шаблон для чекинов вечерних утверждаю

**Comment by Amiditin** [comment]
Пункт про настройку [slash-commands](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/about-slash-commands) добавлен в [конвенцию](https://github.com/atls/convention/wiki/%D0%A2%D1%80%D0%B5%D0%B1%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F-%D0%BA-%D0%BE%D0%BA%D1%80%D1%83%D0%B6%D0%B5%D0%BD%D0%B8%D1%8E#%D0%BD%D0%B0%D1%81%D1%82%D1%80%D0%BE%D0%B9%D0%BA%D0%B0-github)
